html/core/src/jsTest/kotlin/css/StyleSheetTests.kt

```diff
@@ -17,13 +17,13 @@ class StyleSheetTests {
 
     @Test
     fun extendExistingStyleSheet() {
-        val styleSheet = object : StyleSheet(usePrefix = false) {
```

Comment:
let's keep the old constructor test here too. 
Feel free to add a copy of this test with new constructor

Reply:
Fixed :)